patches/minecraft/net/minecraft/client/renderer/entity/RenderItem.java.patch

```diff
@@ -9,6 +9,32 @@
          this.func_175041_b();
      }
  
+@@ -151,7 +151,7 @@
+                 GlStateManager.func_179109_b(-0.5F, -0.5F, -0.5F);
+                 this.func_175036_a(p_180454_2_, p_180454_1_);
+ 
+-                if (p_180454_1_.func_77962_s())
++                if (p_180454_1_.func_77962_s() && net.minecraftforge.client.ForgeHooksClient.renderItemGlint(p_180454_1_, p_180454_2_))
+                 {
+                     this.func_180451_a(p_180454_2_);
+                 }
+@@ -174,14 +174,14 @@
+         float f = (float)(Minecraft.func_71386_F() % 3000L) / 3000.0F / 8.0F;
+         GlStateManager.func_179109_b(f, 0.0F, 0.0F);
+         GlStateManager.func_179114_b(-50.0F, 0.0F, 0.0F, 1.0F);
+-        this.func_175035_a(p_180451_1_, -8372020);
++        this.func_175035_a(p_180451_1_, net.minecraftforge.client.ForgeHooksClient.getGlintColor());
```

Comment:
It would probably be better suited to pass the ItemStack down here as well, yes it's not available in this function but as this is a private function you can pass the value in and keep the patch small with the proper exc/srg lines.